about.html

```diff
@@ -0,0 +1,124 @@
+                <ul>
+                  <li ><a class="menu" href="index.html">Home</a></li>
+                  <li ><a class="menu" href="./about.html">About</a></li>
+                  <li ><a class="menu" href="#about.html">Program</a></li>
+                  <li ><a class="menu" href="#contact-me">Join</a></li>
+                  <li ><a class="menu" href="#about-me">Sponser</a></li>
+                  <li ><a class="menu" href="#contact-me">Register</a></li>
+                  <li ><a href="index.html"> <button class="menu-btn"> Presentation</button></a></li>
+                </ul>
+            </div>
+            </div>
+        
+        <div class="mobile-menu" id="mobileMenu">
+            <img id="menu-close" src="images/closeIcon.png" alt="close icon">
+            <ul class="mob-menu">
+              <li class="menu"><a class="moblink-menu" href="./index.html">Home</a></li>
+              <li class="menu"><a class="moblink-menu" href="#about.html">Program</a></li>
+              <li class="menu"><a class="moblink-menu" href="#contact-me">Join</a></li>
+              <li class="menu"><a class="moblink-menu" href="#about-me">Sponser</a></li>
+              <li class="menu"><a class="moblink-menu" href="#contact-me">Register</a></li>
+            </ul>
+          </div>
+         </div>
+      </nav>
+    
+        <div class="abfront">
+            <div class="abfront1">
+        
+            <h1>SOFTWARE DEVELOPERS <br> GLOBAL SUMMIT 2023</h1>
+           
+            <p> <span> The Creative Commons Global Summit</span> , held every two years, is a place where CC activists from all over the
+                world and experts, scholars and activists from all walks of life who support and practice open culture gather together.</p>
+            </div>
+            <div class="abfront2">
+            <p class="abpbox1">In particular, at this year's CC Summit, we
+                are waiting for the participation of various organizations and individual participants who want to participate in activities to spread the value of sharing, free culture, and open knowledge with CC.
+                Free software supporters, Wikipedia activists, open knowledge foundation activists, art galleries, libraries, museums, archives, governments,
+                foundations, lawyers, social activists, etc. various people active in the field of open content and open knowledge gather in one place to
+                share richer and more dynamic We hope that it will be an opportunity to share our thoughts on creating an ecosystem.
+            </p>
+            <h5 class="class5">June 2023 (Wednesday - Saturday) <br>
+                See you at the Global Summit</h5>
+            </div>
+        </div>
+              <p class="abtpara">Do you have any other questions about software developer Global Summit 2015?</p>
+         
+    </section>
+
+        <section class="logowinner">
+            <h1>Logo for software developmet for 2023</h1>
```

Comment:
**OPTIONAL** _ As I have previously pointed out, please avoid having more than one `h1` tag in an HTML page. Please fix this 👍 👍